frontend/control-center/src/pages/Connectors/Providers/Rasa/RasaConnect.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {useState} from 'react';
+import {useSelector} from 'react-redux';
+import {StateModel} from '../../../../reducers';
+import {Input} from 'components';
+import {ConfigureConnector} from '../../ConfigureConnector';
+import {useTranslation} from 'react-i18next';
+import styles from './RasaConnect.module.scss';
+import {ComponentName, ConnectorName} from 'model';
+
+type RasaConnectProps = {
+  createNewConnection: (url: string, username: string, password: string) => void;
+  isEnabled: boolean;
```

Comment:
aren't `webhookUrl, apiHost, token` the parameters for the `createNewConnection` for Rasa?